amplify/backend/function/getMembersForColony/src/index.js

```diff
@@ -0,0 +1,210 @@
+/**
+ * @type {import('aws-lambda').APIGatewayProxyHandler}
+ */
+
+const { Decimal } = require('decimal.js');
+const { constants, utils } = require('ethers');
+const { getColonyNetworkClient, Network } = require('@colony/colony-js');
+const {
+  providers,
+  utils: { Logger },
+} = require('ethers');
+
+Logger.setLogLevel(Logger.levels.ERROR);
+
+const { calculatePercentageReputation } = require('./reputation');
+const { graphqlRequest } = require('./utils');
+const { getUser, getWatchersInColony } = require('./graphql');
+
+/*
+ * @TODO These values need to be imported properly, and differentiate based on environment
+ */
+const API_KEY = 'da2-fakeApiId123456';
+const GRAPHQL_URI = 'http://localhost:20002/graphql';
+
+const ALL_DOMAIN_ID = 0;
+const ROOT_DOMAIN_ID = 1; // this used to be exported from @colony/colony-js but isn't anymore
+const RPC_URL = 'http://network-contracts.docker:8545'; // this needs to be extended to all supported networks
+const REPUTATION_ENDPOINT = 'http://network-contracts:3002';
+
+// @TODO want to share this enum with the frontend
```

Comment:
Is this comment still relevant? Looks like you have this enum in `generated.ts`. As for sharing it with the function, I don't think it's possible without setting up a function-specific codegen

Reply:
As mentioned in the call, Its defined in schema.graphql, I want to use the enum in the lambda.
`generated.ts` is useful for use in the Cdapp, but currently it seems that redefining the enum as an object in the lamdba func has become the common practice.